### ISSUE TITLE
Stock items are deprecated

### ISSUE THREAD

**Issue by carrotIndustries** [sebp/PyGObject-Tutorial#78]
The file chooser example uses stock items for the open / cancel button, although these are deprecated.

**Comment by sebp** [comment]
Correct, stock icons are deprecated, but they are usually not displayed (as you can see in the screenshots). The advantage is that their text is automatically translated into the user's locale, which otherwise has to be done explicitly, i.e., manual use of gettext.

Stock items are used in multiple examples: filechooserdialog_example.py, dialog_example.py and textview_example.py

**Comment by vanillajonathan** [comment]
So what is the recommended way for dealing with icons?

Lets say I want Cut, copy, and paste or Bold, italic, and underline. Or maybe a settings icon?

**Comment by TingPing** [comment]
@vanillajonathan Depends on the usage, if in a menu probably just don't have an icon but you can pack a box manually, If in a toolbar/header yo use named icons.

**Comment by vanillajonathan** [comment]
http://python-gtk-3-tutorial.readthedocs.io/en/latest/iconview.html
The example calls `Gtk.IconTheme.get_default().load_icon` using `edit-cut`, `edit-paste`, `edit-copy` are these valid names or deprecated?

Where can I find a list of valid icon names that I can use?
It would be nice if I could see the names along with an image so I can see how the icon looks like.
Are named icons deprecated?

**Comment by TingPing** [comment]
`gtk3-icon-browser` is helpful.

**Comment by vanillajonathan** [comment]
Thanks! :)

I would like to request a online web reference of all named icons.

**Comment by TingPing** [comment]
I think recommending `gtk3-icon-browser` is the only option. What icons look like depend upon your theme.
